chrome/scripts.js

```diff
@@ -34,6 +34,26 @@ function runClock(targetDate) {
 var timeinterval = setInterval(runClock, 1000);
 
 // Issue #10: Write html and javascript to display contests from the paramter array
-// @params allContests: array of contest-obeject (refer to backgroung.js)
+// @params allContests: array of contest-obeject (refer to backgroung.js)/
+
 // @return none
-function displayContests() {}
+
+function displayContests(allContests) {
+    const container = document.getElementById("container");
+    allContests.map((contest) => {
+        let element = document.createElement("div");
+        element.className = "contestDiv";
```

Comment:
also create class names for other html elements. In future there will be an open issue to style them and create themes, in which only stylesheet files will be allowed to be edited. So classes need to made beforehand.